internal/pkg/dependencies/common.go

```diff
@@ -79,8 +64,15 @@ func (c *common) StorageApi() (*storageapi.Api, error) {
 		}
 
 		// Token and manifest project ID must be same
-		if c.projectManifest != nil && c.projectManifest.ProjectId() != c.storageApi.ProjectId() {
-			return nil, fmt.Errorf(`given token is from the project "%d", but in manifest is defined project "%d"`, c.storageApi.ProjectId(), c.projectManifest.ProjectId())
+		if c.LocalProjectExists() {
+			prj, err := c.LocalProject()
```

Comment:
V tomto commite je dost zmien, ... su to vacsinou iba zmeny pouzitia metod z `dependencies`.